app/Http/Controllers/ProjectController.php

```diff
@@ -16,31 +17,31 @@ class ProjectController extends Controller
      */
     public function index(Request $request)
     {
-        if ($request->get('pid')) {
-            $projects = Project::find(16);
-            $projects->members->pluck('id')->toArray();
+      if ($request->get('pid')) {
+        $projects = Project::find(16);
+        $projects->members->pluck('id')->toArray();
```

Comment:
The array is never assigned to a variable, so this line seems to have no purpose.

Reply:
This is member variable